src/meetingsession/MeetingSessionConfiguration.ts

```diff
@@ -112,6 +112,12 @@ export default class MeetingSessionConfiguration {
    */
   applicationMetadata: ApplicationMetadata;
 
+  /**
+   * Keep the last frame of the video when a remote video is paused via the pauseVideoTile API.
+   * This is done by skipping the srcObject property of the videoElement.
```

Comment:
> This is done by skipping the srcObject property of the videoElement.

I feel that this is an implementation detail. In WebRTC, pausing a video often means that the user sets `MediaStreamTrack.enabled` to `false`.

Reply:
I think its necessary to expose this implementation detail, given that the `VideoTileController` API includes `HTMLVideoElement`s as arguments.  Builders should have some sort of idea for how we are 'binding' the video element to the 'remote video' (technically `MediaStream` but we hide that).